app/src/main/kotlin/it/vfsfitvnm/vimusic/Database.kt

```diff
@@ -117,6 +117,9 @@ interface Database {
     @Query("SELECT COUNT (*) FROM SearchQuery")
     fun queriesCount(): Flow<Int>
 
+    @Query("UPDATE Format SET isDownloaded = false WHERE songId = :id")
+    fun unMarkDownloaded(id: String)
```

Comment:
let's not create a separate method for this but instead modify existing function:
`fun markDownloaded(songId: String, downloaded: Boolean)`